tests/test_flows.sh

```diff
@@ -203,8 +203,13 @@ test_ap_capability_query() {
     check_error=0
     check send_CAPI_command ${GATEWAY} "DEV_SEND_1905,DestALid,$mac_agent1,MessageTypeValue,0x8001" $redirect
     sleep 1
+    
```

Comment:
Should be a fixup to the commit which added the test and just reorder it to be last

Reply:
Fixed